guide/C/ch_invest.xml

```diff
@@ -1813,6 +1813,840 @@ Income
+              <guilabel>Lots in Account SSSS</guilabel> window where SSSS is the
+              security account.
+            </para>
+
+            <figure pgwide="1">
+              <title>Selling Shares - Capital Gains - Lots before Scrub Account
+              </title>
+              <screenshot id="invest2-lots1B4ScrubAcct">
+                <mediaobject>
+                  <imageobject role="html">
+                  <imagedata fileref="figures/invest2Lots1B4Scrub.png"
+                    format="PNG" srccredit="Chris Good" width="510px"></imagedata>
+                  </imageobject>
+                  <imageobject role="fo">
+                    <imagedata fileref="figures/invest2Lots1B4Scrub.png"
+                               format="PNG" srccredit="Chris Good" ></imagedata>
+                  </imageobject>
+
+                  <textobject>
+                    <phrase>Example of Lots in Account window before Scrub Account
+                      is used.
+                    </phrase>
+                  </textobject>
+
+                  <caption><para>An image of the <emphasis>Lots in Account
+                    </emphasis> window before Scrub Account is used.</para>
+                  </caption>
+                </mediaobject>
+              </screenshot>
+            </figure>
+          </listitem>
+
+          <listitem>
+            <para>Click the <guilabel>Scrub Account</guilabel> button.</para>
+
+            <figure pgwide="1">
+              <title>Selling Shares - Capital Gains - Lots after Scrub Account
+              </title>
+              <screenshot id="invest2-lots2AftScrubAcct">
+                <mediaobject>
+                  <imageobject role="html">
+                  <imagedata fileref="figures/invest2Lots2LotsAftScrubAcct.png"
+                    format="PNG" srccredit="Chris Good" width="510px"></imagedata>
+                  </imageobject>
+                  <imageobject role="fo">
+                    <imagedata fileref="figures/invest2Lots2LotsAftScrubAcct.png"
+                               format="PNG" srccredit="Chris Good" ></imagedata>
+                  </imageobject>
+
+                  <textobject>
+                    <phrase>Example of Lots in Account window after using Scrub
+                      Account
+                    </phrase>
+                  </textobject>
+
+                  <caption><para>An image of the <emphasis>Lots in Account
+                    </emphasis> window after using Scrub Account.</para>
+                  </caption>
+                </mediaobject>
+              </screenshot>
+            </figure>
+
+            <note>
+              <para>After using the <guilabel>Scrub Account</guilabel> button only
+                the last lot is shown, so the above image is after the <guilabel>
+                Lots in Account</guilabel> window has been closed and reopened so
+                all the lots show.
```

Comment:
Are you sure this is correct?  I just tried it and after Scrub Account I saw all the lots in the lot viewer.  If what you say is correct, it sounds like a bug that should be fixed instead of documented.